packages/sanity/src/core/versions/components/dialog/CreateBundleDialog.tsx

```diff
@@ -20,10 +25,26 @@ export function CreateBundleDialog(props: {
   })
 
   const handleOnSubmit = useCallback(
-    () => (bundle: Bundle) => {
-      onSubmit(bundle)
+    async (bundle: Bundle) => {
+      setIsCreating(true)
+      try {
+        await createBundle({
```

Comment:
Adding in creation to the dialog, as figuring all places where this dialog is used will want identical behavior on bundle form submit. `onCreate` still called after the fact.

Also thinking this might be more ideal to just pass an `open` prop to this dialog and have it control whether it is visible